steam_shortcuts.py

```diff
@@ -232,16 +233,15 @@ def get_library_folders(steam_path, library_index_path):
     Returns the library locations
     """
 
-    locations = []
+    libraries = []
 
-    # Find the lines matching a library folder (the library index, and the location)
-    p = re.compile('"\d{1,3}".+".+"')
-    with open(library_index_path.resolve(), encoding="utf-8") as index:
-        locations = [p.findall(line.strip()) for line in index if p.search(line)]
+    libVDF = vdf.load(open(library_index_path))
```

Comment:
Let's keep the `with` block. Also, I would prefer keeping with `snake_case` style, e.g. `lib_vdf = ...`
```suggestion
    with open(library_index_path) as index_file:
        lib_vdf = vdf.load(index_file)
```